.kokoro/docker/docs/Dockerfile

```diff
@@ -1,16 +1,8 @@
 # Copyright 2020 Google LLC
```

Comment:
@vi3k6i5 This should fix the keyserver error (the change in https://github.com/googleapis/synthtool/pull/1155). 

Sorry about sitting on this PR, I completely forgot about it 😅.

Owlbot should open PRs automatically if any of these files change at the template source (synthtool).

Reply:
No worries, Thanks for fixing the issues :)